frontend/src/test/store.js

```diff
@@ -31,7 +31,7 @@ export function createReduxStore(initialState = {}) {
 export function mountComponentWithStore(Component, store, props = {}) {
     return mount(
         <Provider store={store}>
-            <ConnectedRouter history={history}>
+            <ConnectedRouter history={history} noInitialPop>
```

Comment:
This prop is required to omit an initial navigation dispatch from the router, which could have side-effects like resetting some initial state passed to the root reducer factory function. Please check the comment in _PluralSelector.test.js_ for further details.

Reply:
Here again, can you write this down in the code too? Thanks!

Reply:
Done as well 👍